### DISCUSSION TITLE
Please add Fx tv (Persian movie (

### DISCUSSION THREAD

**Discussion by eye2eyeam** [iptv-org/community#98] [Show and tell]
[Stream Link](http://fxtv.wns.live/) )

**Comment by Sphinxroot** [comment]
http://51.210.199.52/hls/stream.m3u8

**Comment by freearhey** [comment]
Added.

https://github.com/iptv-org/iptv/blob/ba0962ec4e746d8ec2e12398ee89642ffeea710d/streams/ir_wnslive.m3u#L6-L7
